graviti/manager/branch.py

```diff
@@ -55,7 +55,9 @@ class BranchManager:
     def __init__(self, dataset: "Dataset") -> None:
         self._dataset = dataset
 
-    def _generate(self, offset: int = 0, limit: int = 128) -> Generator[Branch, None, int]:
+    def _generate(
+        self, offset: Optional[int] = None, limit: Optional[int] = None
```

Comment:
shall we add default for `_generate` function?
will it better to remove the default value for `_generate`?